features/discovery/src/main/resources/META-INF/opennms/applicationContext-discovery.xml

```diff
@@ -56,16 +56,6 @@
     <constructor-arg ref="eventForwarder" />
   </bean>
 
-  <bean id="activemq" class="org.apache.activemq.camel.component.ActiveMQComponent">
```

Comment:
We removed the activemq bean here, but we're still referencing it in the Camel context.